lib/modules/manager/gitlabci/utils.ts

```diff
@@ -24,14 +24,32 @@ const depProxyRe = regEx(
  * @param imageName as used in .gitlab-ci.yml file
  * @return package dependency for the image
  */
-export function getGitlabDep(imageName: string): PackageDependency {
+export function getGitlabDep(
+  imageName: string,
+  registryAliases?: Record<string, string>
+): PackageDependency {
   const match = depProxyRe.exec(imageName);
   if (match?.groups) {
     const dep = { ...getDep(match.groups.depName), replaceString: imageName };
     // TODO: #7154
     dep.autoReplaceStringTemplate = `${match.groups.prefix}${dep.autoReplaceStringTemplate}`;
     return dep;
-  } else {
-    return getDep(imageName);
   }
+
+  for (const [name, value] of Object.entries(registryAliases ?? {})) {
```

Comment:
looks like we can shift this to the `getDep` function, so we can easily extend to more docker images?

Reply:
@viceice Do you mean moving the entire `registryAliases` logic there? Should we then support it properly there (including docs)?

Reply:
yes, move it there but in this PR only pass the aliases from this manager. so we can add more managers in a future pr. as long as the aliases are optional and only passed from gitlabci manger, we don't need to update other docs.